src/components/templates/Poster/components/IntroVideoPreviewModal/IntroVideoPreviewModal.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Modal from "react-bootstrap/Modal";
+
+import { PosterVenue } from "types/venues";
+
+import { WithId } from "utils/id";
+
+import "./IntroVideoPreviewModal.scss";
+
+export interface IntroVideoPreviewModalProps {
+  isVisible: boolean;
+  onHide: () => void;
+  posterVenue?: WithId<PosterVenue>;
+}
+
+export const IntroVideoPreviewModal: React.FC<IntroVideoPreviewModalProps> = ({
+  isVisible,
+  onHide,
+  posterVenue,
+}) => {
+  return (
+    <Modal
+      show={isVisible}
+      onHide={onHide}
+      dialogClassName="IntroVideoPreviewModal"
+      centered
+      size="lg"
+      modal
+    >
+      {posterVenue && (
+        <IntroVideoPreviewModalContent posterVenue={posterVenue} />
+      )}
+    </Modal>
+  );
+};
+
+interface IntroVideoPreviewModalContentProps {
+  posterVenue: WithId<PosterVenue>;
+}
+
+const IntroVideoPreviewModalContent: React.FC<IntroVideoPreviewModalContentProps> = ({
+  posterVenue,
+}) => {
+  return (
+    <>
+      <p className="IntroVideoPreviewModal__title">{posterVenue.title}</p>
+      <iframe
+        className="IntroVideoPreviewModal__iframe"
+        title="IntroVideoPreviewModal-iframe"
+        src={posterVenue.introVideoUrl}
+      />
+    </>
```

Comment:
Because the content here is so basic, it kind of feels like overkill to extract a separate content sub-component. I believe the other place we did this had more complex hooks/logic/etc to conditionally run; not just plain JSX.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1299/commits/7ed5566a94dbad2c9f2e88d2d603ab628a00a83d